website/source/docs/secrets/rabbitmq/index.html.md

```diff
@@ -0,0 +1,326 @@
+---
+layout: "docs"
+page_title: "Secret Backend: RabbitMQ"
+sidebar_current: "docs-secrets-rabbitmq"
```

Comment:
In fact you can try running the local website server and actually test this to avoid surprises.

Have `Ruby 2.x`, and `bundler` installed.
Then run the following:
cd website
`bundle`
`bundle exec middleman server`